services/blockchain-connector/shared/sdk/genesisBlock.js

```diff
@@ -80,9 +84,41 @@ const getGenesisConfig = async () => {
 	}
 };
 
+const getPoSGenesisStakers = async () => {
```

Comment:
This should be implemented in a generic manner where the genesis block is fetched via the getBlock or getGenesisBlock method. Let's discuss this offline.